packages/lit-html/src/hydrate.ts

```diff
@@ -95,7 +95,7 @@ type ChildPartState =
  *                                 # by render(). Includes the digest of the
  *                                 # template
  *   <div class="TEST_X">
- *     <!--lit-bindings 0--> # Indicates there are attribute bindings here
+ *     <!--lit-node 0--> # Indicates there are attribute bindings here
```

Comment:
Let's rename this file `experimental-hydrate.js`

Reply:
Done in https://github.com/Polymer/lit-html/pull/1743